Bundling Error using Webpack

**First of all, thanks for this great project!** 👏

--- 

When trying to include this project into a preact-cli based PWA, I got the following error:

```sh
Build failed!

× ERROR ../node_modules/opencv-ts/src/opencv.js
Module not found: Error: Can't resolve 'fs' in '.\node_modules\opencv-ts\src'
 @ ../node_modules/opencv-ts/src/opencv.js 8:1395-1408 8:8592374-8592387     
 @ ../node_modules/babel-loader/lib??ref--4!./routes/home/index.tsx
 @ ./routes/home/index.tsx
 @ ./components/app.tsx
 @ ./index.ts
 @ ../node_modules/preact-cli/lib/lib/entry.js
```

After a bit of searching for this error I stumbled upon https://github.com/tensorflow/tfjs/issues/1274, with a proposed fix of adding
```json
"browser": {
    "fs": false
}
```
to `package.json`. I tested it and it works, I am gonna open a PR in a minute. 👌

--- a/package.json
+++ b/package.json
@@ -17,5 +17,8 @@
     "repository": {
         "type": "git",
         "url": "https://github.com/theothergrantdavidson/opencv-ts"
+    },
+    "browser": {
+        "fs": false
     }
 }

--- a/package.json
+++ b/package.json
@@ -1,6 +1,6 @@
 {
     "name": "opencv-ts",
-    "version": "1.2.5",
+    "version": "1.2.6",
     "main": "src/opencv.js",
     "types": "src/opencv.d.ts",
     "description": "This package is a work in progress to translate opecv with a typescript definition",

